packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -244,13 +253,14 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
   };
 
   private _onAChanged = (ev: React.MouseEvent<HTMLElement>, a: number): void => {
-    this._updateColor(ev, updateA(this.state.color, Math.round(a)));
+    this._updateColor(ev, updateA(this.state.color, this._getSliderValue(Math.round(a))));
   };
 
   private _onTextChange(component: keyof IRGBHex, event: React.FormEvent<HTMLInputElement>, newValue?: string): void {
     const color = this.state.color;
     const isHex = component === 'hex';
     const isAlpha = component === 'a';
+    newValue = isAlpha ? String(this._getSliderValue(Number(newValue))) : newValue;
```

Comment:
You should do it later in code since we have not validated that newValue is a valid number

Reply:
Done.